website/docs/r/private_endpoint.html.markdown

```diff
@@ -234,10 +234,12 @@ A `private_service_connection` block supports the following:
 
 * `private_connection_resource_alias` - (Optional) The Service Alias of the Private Link Enabled Remote Resource which this Private Endpoint should be connected to. One of `private_connection_resource_id` or `private_connection_resource_alias` must be specified. Changing this forces a new resource to be created.
 
-* `subresource_names` - (Optional) A list of subresource names which the Private Endpoint is able to connect to. `subresource_names` corresponds to `group_id`. Possible values are detailed in the product [documentation](https://docs.microsoft.com/azure/private-link/private-endpoint-overview#private-link-resource) in the `Subresources` column. Changing this forces a new resource to be created.
+* `subresource_names` - (Required) A list of subresource names which the Private Endpoint is able to connect to. `subresource_names` corresponds to `group_id`. Possible values are detailed in the product [documentation](https://docs.microsoft.com/azure/private-link/private-endpoint-overview#private-link-resource) in the `Subresources` column. Changing this forces a new resource to be created. 
 
 -> **NOTE:** Some resource types (such as Storage Account) only support 1 subresource per private endpoint.
 
+-> **NOTE:** Only for Private-link resource called "Private Link service", `subresource_names` can be omitted.
```

Comment:
Perhaps we can turn this around to better help users understand they need to specify values in most cases?
```suggestion
-> **NOTE:** For most Private Links one or more `subresource_names` will need to be specified, please see the linked documentation for details.
```

Reply:
Hi @jackofallops, now that I have a better understanding I'll go with your suggestion.